### PR TITLE
Md format fix

### PR DESCRIPTION
**Pull request by glicht**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
* support `<hr>` and  `<br></br>` cases in md
* add pyupgrade to dev reqs (for some reason I needed this to get it running)

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -3,5 +3,6 @@ pre-commit
 pytest
 pytest-cov
 pytest-mock
+pyupgrade
```
Comment (Itay4) [inline requirements-dev.txt:6]: no need to specify version ? we use v2.3.0 in the pre commit hooks

File: demisto_sdk/commands/generate_docs/common.py [hunk 2]
```diff
@@ -266,10 +266,13 @@ def format_md(md: str) -> str:
     :return:
```
Comment (Itay4) [inline demisto_sdk/commands/generate_docs/common.py:266]: update docstring please

File: requirements-dev.txt (hunk 1 above)
Comment (glicht) [reply requirements-dev.txt]: after changing to 2.4.1 it is working. not sure. anyway removed this from dev reqs

File: demisto_sdk/commands/generate_docs/common.py (hunk 2 above)
Comment (glicht) [reply demisto_sdk/commands/generate_docs/common.py:266]: done

**Comment by Itay4** [comment]
@glicht can we merge this ?

**Pull request merged**